tasks/task07/order_repository.py

```diff
@@ -0,0 +1,119 @@
+#Task 07: OrderRepository
+import time
+class Good:
+    def __init__(self, id:str = "", name: str = "", price = 0):
+        self.id = id 
+        self.name = name
+        self.price = price
+    def __str__(self):
+        return f"id: {self.id}\tname: {self.name}\tprice: {self.price}"
+
+class Order:
+    def __init__(self, order_id, order_date, client_id, goods: list[Good] = []):
+        self.order_id = order_id
+        self.order_date = order_date
+        self.client_id = client_id
+        self.goods = goods
+        self.price = 0
+        for igood in goods:
+            self.price += igood.price
+    def __str__(self):
+        res = f"order_id: {self.order_id}\torder_date: {time.strftime('%Y-%m-%d_%H:%M.%S',time.gmtime(self.order_date))}\tclient_id: {self.client_id}\tprice: {self.price}\n\tGoods:\n"
+        for igood in self.goods:
+            res += "\t" + str(igood) + "\n"
+        return res
+
+
+class OrderRepository:
+    def __init__(self, orders: list[Order] = []):
+        self.orders = orders
+    def __str__(self):
+        res = ""
+        for iorder in self.orders:
+            res += str(iorder)
+        return res
+    def add(self, order: Order):
+        self.orders.append(order)
+        return 0
```

Comment:
In Python, just do not return anything if your function has no return value. Do not write `return` at all.

FYI: If function has no `return`, it implicitly returns `None`